Makefile.kokkos

```diff
@@ -1399,22 +1392,12 @@ ifeq ($(KOKKOS_INTERNAL_USE_CUDA), 1)
 else
   tmp := $(call desul_append_header,"/* $H""undef DESUL_ATOMICS_ENABLE_CUDA */")
 endif
-ifeq ($(KOKKOS_INTERNAL_CUDA_USE_RELOC), 1)
-  tmp := $(call desul_append_header,"$H""define DESUL_ATOMICS_ENABLE_CUDA_SEPARABLE_COMPILATION")
-else
-  tmp := $(call desul_append_header,"/* $H""undef DESUL_ATOMICS_ENABLE_CUDA_SEPARABLE_COMPILATION */")
-endif
```

Comment:
I suppose you didn't mean to remove this

Reply:
Oh yes, I think somehow my branch just missed those options. I replaced it again. Thanks.